src/hcli/commands/plugin/search.py

```diff
@@ -110,7 +110,7 @@ def handle_plugin_name_query(plugins: list[Plugin], query: str, current_version:
     plugin = get_plugin_by_name(plugins, query)
     latest_metadata = get_latest_plugin_metadata(plugin)
 
-    metadata_dict = latest_metadata.plugin.model_dump()
+    metadata_dict = latest_metadata.plugin.model_dump(by_alias=True)
```

Comment:
good catch. i'm concerned that our tests didn't catch this, so i'd like to add a test case, too. I don't mind doing that, and I'll take care of it shortly.